### PR TITLE
add payment types table to compare page

### PR DESCRIPTION
**Pull request by cathydeng**
## Changes
adding first checkbox table - payment types accepted

## Screenshots
Screen Shot 2022-11-04 at 12 35 44 PM
Screen Shot 2022-11-04 at 12 36 00 PM

## Checklist

- [ ] if adds a new page, adds accessibility tests
- [ ] if adds a new page or new interaction, adds e2e test
- [ ] if adds a new page or new interaction, sends google analytics events and updates [GA doc](https://docs.google.com/document/d/1DfQDUNZPO3B352EhUwXp0el6qrAWz8Jq9cTa1yv8Ty4/edit)
- [ ] if changes filter functionality, updates [filters doc](https://docs.google.com/document/d/1yEdo7IpHCtEKNNF6FMLapBUgcPw_8CY7wGwxKTdtJrU/edit)

Fixes https://app.asana.com/0/0/1203186341961985/f

### CODE REVIEW

File: src/components/Compare/CompareStripedRows.tsx [hunk 1]
```diff
@@ -11,8 +11,22 @@ function CompareStripedRows({
...
 }) {
+  const getTableCell = (value: string | boolean) => {
+    if (value === true) {
+      return (
+        <div className="display-flex flex-justify-center">
+          <Check />
+        </div>
+      );
+    } else if (value === false) {
+      return null;
```
Comment (cathydeng) [inline src/components/Compare/CompareStripedRows.tsx:22]: the checkmarks might be an odd experience on screen readers...open to ideas on making this more screen-reader-friendly, e.g. having it say something explicit when there is a checkmark + absence of checkmark? cc @admoorgit

Comment (jlhogan) [reply src/components/Compare/CompareStripedRows.tsx:22]: I think we should add aria-labels to the checkboxes to say something to the effect of "[feature name] present at this location" / "[feature name] not present at this location"

**Review by jlhogan — APPROVED** [review]
LGTM, nit that the icons look like they have a bit more space on the bottom vs top, i think they could use a tiny bit more space/padding on the top to match

File: src/components/Compare/CompareStripedRows.tsx (hunk 1 above)
Comment (cathydeng) [reply src/components/Compare/CompareStripedRows.tsx:22]: cool - waiting on adam to confirm language, will merge this and then we can have additional tweaks in a separate PR

**Pull request merged**